src/vault.ts

```diff
@@ -45,16 +45,21 @@ function findAnsibleConfigurationFile(vaultFile: Uri): Promise<Uri> {
   })
 }
 
-async function parseVaultPasswordFilePath(ansibleCfgFile: Uri) {
-  const content = await readFile(ansibleCfgFile.path)
-  const passwordFile = content.match(/^\s*vault_password_file\s*=\s*(.*?)\s*$/m)
-  if (passwordFile) {
-    const vaultPasswordFile = passwordFile[1]
-    const ansibleCfgDirectory = dirname(ansibleCfgFile.path)
-    const fullPath = resolve(ansibleCfgDirectory, vaultPasswordFile)
-    log.appendLine(`Found vault_password_file in ${ansibleCfgFile.path}, resolved ${vaultPasswordFile} to ${fullPath}`)
-    return Uri.parse(fullPath)
-  }
+async function findVaultPasswordFilePath(ansibleCfgFile: Uri) {
+  if (process.env.ANSIBLE_VAULT_PASSWORD_FILE) {
+      progress.report({ message: 'ANSIBLE_VAULT_PASSWORD_FILE is set in environment, awesome!' })
```

Comment:
This doesn't compile, as `progress` isn't available in the scope. You can run the plugin from VSCode by pressing F5 and test this.